modules/gmc-agent/src/main/java/org/gridgain/service/tracing/GmcSpanExporter.java

```diff
@@ -63,6 +72,12 @@
     /** Exporter. */
     private OpenCensusTraceExporter exporter;
 
+    /** Executor service. */
+    private final ExecutorService exSrvc = Executors.newSingleThreadExecutor();
```

Comment:
Why executor not incapsulated in sender?

Reply:
Agree, move to sender.